yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -550,7 +551,15 @@ export class AztecNodeService implements AztecNode {
   public async simulatePublicCalls(tx: Tx) {
     this.log.info(`Simulating tx ${await tx.getTxHash()}`);
     const blockNumber = (await this.blockSource.getBlockNumber()) + 1;
-    const newGlobalVariables = await this.globalVariableBuilder.buildGlobalVariables(new Fr(blockNumber));
+
+    // TODO(benesjan): populate these values from config.
```

Comment:
Does the todo have an issue or will it be part of this one?

Reply:
Forgot about this one. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4421/commits/61d7839106ce083a6fe659aecdd5784f480963a4 Thanks